index.html

```diff
@@ -1,270 +1,341 @@
+                    <li class="about-text2 ">CSS</li>
+                </ul>
+            </div>
+
+            <div class="about-card ">
+                <img class="about-icon " src='./Images/icon-frameworks.png' alt="Language logo ">
+                <p class="about-title ">
+                    Frameworks
+                </p>
+
+                <ul class="about-Languages ">
+                    <li class="about-text2 ">Bootstrap</li>
+                    <li class="about-text2 ">Ruby on rails</li>
+                    <li class="about-text2 ">RSpec</li>
+                    <li class="about-text2 ">Capybara</li>
+                    <li class="about-text2 ">Selenium</li>
+                </ul>
+            </div>
+            <div class="about-card ">
+                <img class="about-icon " src='./Images/icon-skills.png' alt="Language logo ">
+                <p class="about-title ">
+                    Skills
+                </p>
+
+                <ul class="about-Languages ">
+                    <li class="about-text2 ">Database Managment</li>
+                    <li class="about-text2 ">Version Control</li>
+                    <li class="about-text2 ">CLI</li>
+                    <li class="about-text2 ">Web Development</li>
+                    <li class="about-text2 ">API Design</li>
+                </ul>
+            </div>
+        </div>
+    </section>
+
+    <section" id="contact">
+        <form action="https://formspree.io/f/mqkoyljj" method="post" class="contact-form">
+            <div class="contact-text-desc">
+                <h1 class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat
+                    please get in touch.</h1>
+            </div>
+
+            <div class="form-desc">
+                <ul class="form-ul">
+                    <li class="name-box">
+
+                        <input type="text" value="" name="User_name" placeholder="Full name" class="name" required
+                            maxlength="30">
+
+                    </li>
+                    <li class="email-box">
+
+                        <input type="email" value="" name="User_email" placeholder="Email address" class="email"
+                            required>
+
+                    </li>
+                    <li class="textarea-box">
+
+                        <textarea name="" id="" cols="30" rows="10" name="User_message"
+                            placeholder="  Hey I'd like to connect to.." class="textarea" required
+                            maxlength="500"></textarea>
+
+                    </li>
+
+                </ul>
+                <div class="last-b">
+                    <button class="get-button">
+                        Get in touch
+                    </button>
+                </div>
+
+
+
+                <p class="email1">titelunda@gmail.com</p>
+            </div>
+        </form>
+        </section>

```

Comment:
The contact text is suppose to be at the left side with the contact form to the right. However, 

- [ ] I see your design does not stay that way and tends to shift position as the width changes in size.
- [ ] Part of it also dissappears to the left side. 
 Kindly fix these issues and ensure it matches the actual design for all screen sizes of the desktop version. :pray: 

### Your design 

![Screenshot from 2023-04-07 16-21-19](https://user-images.githubusercontent.com/65233860/230647241-e652904e-50df-4714-a973-57cbc3d762b0.png)



### Figma design 

![Screenshot from 2023-04-07 16-24-11](https://user-images.githubusercontent.com/65233860/230647293-cc762be5-9a0c-43eb-8694-418c0e6bc0b8.png)